index.html

```diff
@@ -1,8 +1,100 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang = "en">
   <head>
-    <meta charset="UTF-8">
+    <meta charset = "UTF-8">
+    <meta name = "viewport" content = "width = device-width, initial-scale = 1">
     <title>Fork Me! FCC: Test Suite Template</title>
-    <link rel="stylesheet" href="css/style.css">
+    <link rel = "stylesheet" href = "css/style.css">
   </head>
+
+  <body>
+    <h1 id = 'title'>freeCodeCamp Survey Form</h1>
+    <p>Thank you for taking the time to help us improve the platform</p>
```

Comment:
- [] You need use semantic tags in your project. Instead of placing 'h1' and 'p' tags freely here, you need to use 'header' tag and place 'h1' and 'p' tags inside the header.